frontend/views/containers/chatroom/SendArea.vue

```diff
@@ -615,38 +612,41 @@ export default ({
         const fileUrl = URL.createObjectURL(file)
         const fileSize = file.size
 
-        if (fileSize > Math.pow(10, 9)) {
-          // TODO: update Math.pow(10, 9) above with the value delivered from the server once it's implemented there.
+        if (fileSize > CHAT_ATTACHMENT_SIZE_LIMIT) {
           return sbp('okTurtles.events/emit', OPEN_MODAL, 'ChatFileAttachmentWarningModal', { type: 'large' })
         } else if (!fileExt || !CHAT_ATTACHMENT_SUPPORTED_EXTENSIONS.includes(fileExt)) {
           // Give users a warning about unsupported file types
           return sbp('okTurtles.events/emit', OPEN_MODAL, 'ChatFileAttachmentWarningModal', { type: 'unsupported' })
         }
 
+        // !@#
         list.push({
           url: fileUrl,
           name: file.name,
           extension: fileExt,
-          attachType: file.type.match('image/') ? 'image' : 'non-image'
+          mimeType: file.type || '',
+          attachmentId: randomHexString(15),
```

Comment:
@corrideat thoughts on this `attachmentId`? Is it fine? Is it long enough? Should it be something else?

Reply:
I had a conversation with @corrideat about this and he says that it seems to be fine here as it's just used for identifying file previews in the UI as ephemeral data. However it shouldn't be included in the chain ([see this comment](https://github.com/okTurtles/group-income/pull/1877#discussion_r1515958005)), and to uniquely identify files, the `manifestCid` is what should be used.